### DISCUSSION TITLE
blog/hands-on-ml-review/

### DISCUSSION THREAD

**Discussion by giscus** [Jitesh117/blog_comments#11] [Q&A]
# blog/hands-on-ml-review/

I’ve heard so much about “Hands-On Machine Learning” as the go-to book for beginners in ML. I first picked it up in January but didn’t get far. Now, though, I’m giving it another shot and committing to finishing it. Today, on June 8th, I’m starting to document this journey, which will end up in a blog post review of the book that you’re reading now :)
Jumping Right In What’s super cool is that the author doesn’t waste time spoon-feeding you theory.

https://jitesh117.github.io/blog/hands-on-ml-review/

**Comment by ramanuj07** [comment]
Great review, thanks for this :)

**Comment by hyperdevstuff** [comment]
Thanks for the Great review!!
I was bit hesitant before that directly moving to ML will be tough but now seeing people learning maths along with ML , a top down approach instead  of ground up really pumped me.
Will try to finish this book before new year arrives..
once again thanks for review.

**Reply by Jitesh117** [reply]
glad you liked it! and thanks for the kind words.
